bootstrap/kubeadm/controllers/kubeadmconfig_controller.go

```diff
@@ -73,6 +73,8 @@ type InitLocker interface {
 type KubeadmConfigReconciler struct {
 	Client           client.Client
 	KubeadmInitLock  InitLocker
+	
+	// WatchFilterValue is the label value used to filter events prior to reconciliation
```

Comment:
```suggestion
	// WatchFilterValue is the label value used to filter events prior to reconciliation.
```
nit (also in the other cases)

Usually our comments end with a dot. Unfortunately, we currently cannot enable the linter for it.